sdk/servicebus/service-bus/src/util/errors.ts

```diff
@@ -253,10 +257,21 @@ export function throwIfNotValidServiceBusMessage(
   msg: unknown,
   errorMessageForWrongType: string
 ): void {
-  if (!isServiceBusMessage(msg)) {
+  if (!isServiceBusMessage(msg) && !isAmqpAnnotatedMessage(msg)) {
     throw new TypeError(errorMessageForWrongType);
   }
-  if (msg.partitionKey && msg.sessionId && msg.partitionKey !== msg.sessionId) {
-    throw new TypeError(PartitionKeySessionIdMismatchError);
+
+  if (isServiceBusMessage(msg)) {
+    if (msg.partitionKey && msg.sessionId && msg.partitionKey !== msg.sessionId) {
+      throw new TypeError(PartitionKeySessionIdMismatchError);
+    }
   }
 }
+
+/** @internal */
+export const errorInvalidMessageTypeSingleOrArray =
```

Comment:
This is used in the sendMessages method and so should also mention ServiceBusMessageBatch as a valid input

Reply:
This error is not thrown when a batch is provided.
![image](https://user-images.githubusercontent.com/10452642/113760303-ea190300-96ca-11eb-8636-e09417088e07.png)

However, it might make sense to have SBMessageBatch in the error message.

Reply:
I have added it, let me know how it looks.